modules/ROOT/pages/notifications/index.adoc

```diff
@@ -78,4 +246,30 @@ The following categories exist, ordered by severity:
 | INFORMATION
 | The query uses costly operations and might be slow. Consider if it is possible to rewrite the query in a different way.
 |
-|===
\ No newline at end of file
+|===
+
+[[server-driver-compatibility]]
+== Server-driver compatibility
+
+The server and driver communicate with each other through the Bolt protocol.
+During the handshake process, they agree on using the newest possible Bolt protocol version that both the server and the driver support.
+For more information on the Bolt versions supported by different server versions, see the link:https://neo4j.com/docs/bolt/current/bolt-compatibility[Bolt Protocol documentation].
+
+The GqlStatusObject API is available in Neo4j 5.22 and later versions on the server side and in the 5.23 driver and later versions on the driver side.
+The current Notification API is still present, and the GqlStatusObject API can be used in parallel with it.
+
+To receive the Gql-status notifications, both your server and the driver must support the GqlStatusObject API.
```

Comment:
```suggestion
To use the GqlStatusObject API, both your server and the driver must support it.
```

Although that is not technically true as you can use it 
1. directly through the server API
2. using a new driver and an old server and let the driver polyfill as described below.

I think the vast majority of our users use Neo4j with a driver though and scenario 2 is not super helpful in comparison with if the server was newer. Maybe we can phrase it like you must have support on both sides to fully utilize the API or something?